test_data/import_order/expected_representation.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Placeholder_1 do
+  alias X.{K, K.T, Y}
+  alias X.{Z}
+  alias X.Y.{A, Z}
+  alias X.Y.{A, Z}
+  alias Banana, as: Placeholder_2
+  alias X
+  alias X.Y
+  alias X.Y.Z
+  alias Z
+  use Z
+  require C
+  import A
+  import B, only: [b: 0]
+  import D
+
+  def placeholder_3(placeholder_4) do
+    placeholder_4 * 2
+  end
+
+  import C, only: [bar: 1]
+  require L
+
+  def placeholder_5() do
+    alias U
```

Comment:
@angelikatyborska I know you said there is not much point ordering the imports in a function, and I agree, but it basically came for free, so why not.